src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldPageTreeNode.class.php

```diff
@@ -352,4 +355,10 @@ private function getUrlUtilService(): UrlUtil
     {
         return ServiceLocator::get('chameleon_system_core.util.url');
     }
+
+    private function getTranslationService(): ChameleonTranslator
```

Comment:
That would be TranslatorInterface and "translator" as id.

However I would all translation changes out of this fix: There are extremely many of those calls and it makes probably sense to replace them in one go in the Core.

Reply:
I removed the translation refactoring.